x/proof/keeper/query_proof_test.go

```diff
@@ -133,7 +133,7 @@ func TestProofQuerySingle(t *testing.T) {
 }
 
 func TestProofQueryPaginated(t *testing.T) {
-	keeper, ctx := keepertest.ProofKeeper(t, nil)
+	keeper, ctx := keepertest.ProofKeeper(t)
```

Comment:
NIT: Do you know why we use `k` in some tests and `keeper` in others? Consistency would be preferable but not a blocker.

Reply:
The `k` variable is the name that was generated by ignite; there are a decent number of tests that we haven't modified since generation which use it.